app/templates/dataset.html

```diff
@@ -200,6 +201,16 @@ <h5><strong> {{ data.title }} </strong></h5>
             <th scope="row">Spatial Coverage:</th>
             <td>{{metadata.spatialCoverage|join(", ")}}</td>
           </tr>
+          {% endif %} {% if metadata.dates is not none %}
+          <tr>
+            <th scope="row">Other Dates:</th>
+            <td>
+              {% for key, value in metadata.dates.items() %}
+              <i>{{ key }}:</i> {{ value }}
+              {{" -- " if not loop.last else "" }}
+              {% endfor %}
+            </td>
+          </tr>>
```

Comment:
I don't think the extra closing bracket here should be here.

Reply:
ooops. Fixed now. Thank you @tkkuehn !!